roles/openshift_metrics/defaults/main.yaml

```diff
@@ -4,8 +4,14 @@ image_version: latest
 master_url: https://kubernetes.default.svc.cluster.local
 
 hawkular_user_write_access: False   #if user accounts should be able to write metrics
+hawkular_cassandra_nodes: 1
+# emptydir|pv|dynamic
+hawkular_cassandra_storage_type: emptydir
+hawkular_cassandra_pv_prefix: metrics-cassandra
+hawkular_cassandra_pv_size: 10Gi
 
 heapster_standalone: False
+heapster_allowed_users: system:master-proxy
```

Comment:
this should be a comma delimited string correct or is only one user allowed for Heapster? 
I haven't checked the README yet, but we should make sure that's explained otherwise someone may use an array (not sure if that'd still work in secret generation)